### DISCUSSION TITLE
Mixing the starter and the main theme repo contents

### DISCUSSION THREAD

**Discussion by Hrovatin** [cotes2020/jekyll-theme-chirpy#2035] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I would like to customize my sidebar and tab name so that different title strings are used in both (adding line break in sidebar but not in tab). If I understand correctly, the only way to do that is to copy sidebar _includes file. Since I started with the chirpy starter, I was wondering what would be the recommended approach to do that and what may break in the future when the theme gets updated via the actions?

### Operations you have already tried

/

**Comment by kungfux** [comment, marked as answer]
Please see a similar discussion on renaming tabs: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/1297

That's right, it is possible to override the logic by replacing the chirpy file in the starter. There is a maintenance cost associated with such a change. Every time you upgrade your blog to a newer version of Gem, you may run into unexpected problems and have to redo your customisations. This could happen if the file you are overriding has been modified in Chirpy, so when you upgrade your blog to a newer Gem, you need to keep an eye on the modified files you are overriding.

I'd suggest considering non-maintenance options if you only want to make small changes, such as using CSS to change the look of the sidebar. This does not guarantee correctness after an upgrade, so you will need to build and test locally first. However, it will minimise the number of potential problems.

**Reply by Hrovatin** [reply]
Thank you for the explanation.
